tests/Unit/Evolution/Systems/ScalarWave/CMakeLists.txt

```diff
@@ -13,6 +13,8 @@ set(LIBRARY_SOURCES
   Test_Constraints.cpp
   Test_EnergyDensity.cpp
   Test_Equations.cpp
+  Test_MomentumDensity.cpp
```

Comment:
you sohuld add this file to the `CMakeLists.txt` in the same commit that you add the file. We like to have each commit compile correctly